simulator/conf.py

```diff
@@ -0,0 +1,25 @@
+from eth2spec.utils.ssz.ssz_typing import uint256, uint64, uint8
+from typing import List, Sequence
+from utils import NodeId, SampleId, bytes_to_uint64, uint_to_bytes, hash, ENDIANNESS
+
+DATA_COLUMN_SIDECAR_SUBNET_COUNT = uint8(128)
```

Comment:
these are not configurations but spec related contants. I think they would best be in their own spec implementations.